# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/loader/ModuleLoadRequest.cpp
# Commit: 02269004c755
# Full Hash: 02269004c7554e6972c13ee6d147fc2ddcee34f7
# Author: Yoshi Cheng-Hao Huang <allstars.chh@gmail.com>
# Date: 2025-09-26 06:07 +0000
# Regressor Bug: 1968890
# File Overlap Count: 5
# Overlapping Files: js/loader/ModuleLoadRequest.cpp, js/loader/ModuleLoaderBase.cpp, js/loader/ModuleLoadRequest.h, js/loader/ModuleLoaderBase.h, dom/script/ScriptLoader.cpp
# Description:
#   Bug 1968890 - Part 7: Add Cancel method. r=jonco
# ==============================================================================

diff -r 9e2a3327cd0f -r 02269004c755 js/loader/ModuleLoadRequest.cpp
--- a/js/loader/ModuleLoadRequest.cpp	Fri Sep 26 06:07:44 2025 +0000
+++ b/js/loader/ModuleLoadRequest.cpp	Fri Sep 26 06:07:44 2025 +0000
@@ -74,22 +74,6 @@
   return mModuleScript->HasParseError() || mModuleScript->HasErrorToRethrow();
 }
 
-void ModuleLoadRequest::Cancel() {
-  if (IsCanceled()) {
-    return;
-  }
-
-  if (IsFinished()) {
-    return;
-  }
-
-  ScriptLoadRequest::Cancel();
-
-  mModuleScript = nullptr;
-  mReferrerScript = nullptr;
-  mModuleRequestObj = nullptr;
-}
-
 void ModuleLoadRequest::SetReady() {
   MOZ_ASSERT(!IsFinished());
 